core/lib/generators/mno_enterprise/install/install_generator.rb

```diff
@@ -117,7 +117,11 @@ def notify_about_routes
     mount MnoEnterprise::Engine, at: '/mnoe', as: :mno_enterprise
   end
 
-  root to: redirect('dashboard/')
+  if Settings.dashboard.public_pages.enabled
+    root to: redirect('dashboard/')
+  else
+    root to: redirect('mnoe/auth/users/sign_in')
+  end
```

Comment:
This is weird, it's already done in #425
That's the exact same change, I'm surprised that Github is showing it